generated/nirfmxspecan/nirfmxspecan.proto

```diff
@@ -986,9 +986,7 @@ enum NiRFmxSpecAnAttribute {
   NIRFMXSPECAN_ATTRIBUTE_DPD_PRE_DPD_CARRIER_BANDWIDTH = 1110116;
   NIRFMXSPECAN_ATTRIBUTE_DPD_IQ_ORIGIN_OFFSET_CORRECTION_ENABLED = 1110117;
   NIRFMXSPECAN_ATTRIBUTE_DPD_DVR_NUMBER_OF_SEGMENTS = 1110119;
-  NIRFMXSPECAN_ATTRIBUTE_DPD_DVR_LINEAR_MEMORY_DEPTH = 1110120;
```

Comment:
@reckenro @amstewart Correct if wrong but I believe the removing of enums is fine as clients building before would still just pass the value to our sever and we would just pass along. The issue would be if the meaning of the enum changed in a future update.

Reply:
@astarche

Reply:
Yeah, removing enum values is "source breaking" so any existing clients that use the enum value will be broken when they uptake new protos. Usually when enums like these are removed, they were not working in the first place. So the runtime behavior will change from one type of error to another, which is usually OK. OR we are intentionally breaking  behavior to remove a feature.

If you ever use the enum value to mean something else ... yeah that would be more of a concern for sure. Ideally these are reserved somehow in your metadata.

I've tagged this as source breaking. Can you add to the description a line that some enum values are removed, why they are removed, and what the change in behavior would be (i.e., "new error"). If there is any chance that someone was doing something useful with the values, add a binary-breaking tag as well.